docs/product/components/page-titles.html

```diff
@@ -0,0 +1,148 @@
+---
+
+<section class="stacks-section">
+    {% header h2 | Components %}
+    <div class="overflow-x-auto mb32">
+        <table class="wmn4 s-table s-table__bx-simple">
+            <thead>
+                <tr>
+                    <th scope="col" class="s-table--cell4">Component</th>
+                    <th scope="col">Description</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <th scope="row">Page title</th>
+                    <td>Title of the page. Refer to the <a href="{{ "/content/guidelines/principles" | relative_url }}">content section</a> of Stacks for things like <a href="{{ "/content/guidelines/principles#simple-words" | relative_url }}">word choice</a>, <a href="{{ "/content/guidelines/principles#naming" | relative_url }}">naming</a>, and <a href="{{ "/content/guidelines/grammar-and-mechanics#use-sentence-casing" | relative_url }}">capitalization</a>.</td>
+                </tr>
+                <tr>
+                    <th scope="row">Description <span class="s-label--status">Optional</span></th>
+                    <td>Additional context for content and actions on the page.</td>
+                </tr>
+                <tr>
+                    <th scope="row">Actions <span class="s-label--status">Optional</span></th>
+                    <td>Free-form area to include buttons, filters, or other HTML content.</td>
+                </tr>
+                <tr>
+                    <th scope="row">Breadcrumbs <span class="s-label--status">Optional</span></th>
+                    <td><a href="{{ "/product/components/breadcrumbs" | relative_url }}">Breadcrumbs</a> can be included to provide context for the currently-viewed page.</td>
+                </tr>
+            </tbody>
+        </table>
+    </div>
+    {% header h2 | Admin sections %}
+    <p class="stacks-copy">
+        These components should be used in the admin sections of our private Q&A products.
+    </p>
+    {% header h3 | Base Styles %}
+    <p class="stacks-copy">
+        The base style for pages that don’t need further description beyond the page title and don’t have any actions for filters. 
+    </p>
+    <div class="stacks-preview">
+{% highlight html %}
+<div class="s-page-title">
+    <div>
+        <h1>Page title</h1>
+    </div>
+</div>
+{% endhighlight %}
+        <div class="stacks-preview--example">
+            <div class="s-page-title">
+                <div>
+                    <h1>Page title</h1>
+                </div>
+            </div>
+        </div>
+    </div>
+    {% header h3 | Title with description %}
+    <p class="stacks-copy">
+        A summary of a page’s content. Helpful for large pages with multiple actions.
+    </p>
+    <div class="stacks-preview">
+{% highlight html %}
+<div class="s-page-title">
+    <div>
+        <h1>Page title</h1>
```

Comment:
A few options, all of which work and don't work on some level:

`<div class="s-page-title--title"></div>`
`<div class="s-page-title--actions"></div>`
This might work since the stuff on the right is almost always buttons or filters (but it would limit us to that).

`<div class="s-page-title--left"></div>`
`<div class="s-page-title--right"></div>`
😐

`<div class="s-page-title--primary"></div>`
`<div class="s-page-title--secondary"></div>`
This might work since the stuff on the right is optional, but is really vague and could be confusing if you only see the first div.